packages/remote-react-renderer/src/lib/createFlowRemoteComponentRenderer.tsx

```diff
@@ -5,17 +5,39 @@ import {
 } from "@/lib/propClassifiers";
 import type { RemoteComponentRendererProps } from "@mittwald/remote-dom-react/host";
 import { createRemoteComponentRenderer } from "@mittwald/remote-dom-react/host";
-import { mapEventHandler, Version } from "@mittwald/flow-remote-core";
+import type { EventSerializationMap } from "@mittwald/flow-remote-core";
+import { FlowRemoteElement } from "@mittwald/flow-remote-elements";
+import type { FlowRemoteElementMetaData } from "@mittwald/flow-remote-elements";
 import {
-  FlowRemoteElement,
-  type FlowRemoteElementMetaData,
-} from "@mittwald/flow-remote-elements";
+  mapEventHandler,
+  Version,
+  standard as defaultEventSerializer,
+} from "@mittwald/flow-remote-core";
 import { type ComponentType } from "react";
 import { mapValues } from "remeda";
 
+const customEventMappings: EventSerializationMap = {
+  onPaste: (event: unknown) => {
+    const clipEvent = event as ClipboardEvent satisfies ClipboardEvent;
+
+    const eventClipBoardData = clipEvent.clipboardData;
+    const windowClipBoardData =
+      (window as never) && "clipboardData" in window && window.clipboardData;
+
+    const pastedData: string = (
+      eventClipBoardData || windowClipBoardData
+    ).getData("text");
+
+    return {
+      ...(defaultEventSerializer(event) as object),
+      content: pastedData,
+    };
```

Comment:
I found this on StackOverflow what does make sense to me:

> No, there is no cross-browser support for window.clipboardData. It is only supported by IE. Support for window.clipboardData is generally considered a security issue because it allows every website you visit to read whatever happens to be in your clipboard at the time.


When dropping `window.clipboardData` the implementation can be much easier:

```suggestion
    if (!(event instanceof ClipboardEvent)) {
      return event;
    }

    const pastedData = event.clipboardData?.getData("text");

    return {
      ...(defaultEventSerializer(event) as object),
      content: pastedData,
    };
```